language-ids.sublime-settings

```diff
@@ -26,6 +26,8 @@
     "source.js": "javascript",
     "source.js.react": "javascriptreact", // https://github.com/Thom1729/Sublime-JS-Custom
     "source.jsx": "javascriptreact",
+    "source.fixedform-fortran": "fortran", // https://packagecontrol.io/packages/Fortran
```

Comment:
Please keep this list sorted.

Reply:
fixed. :)

Was the `languageId` deprecated?